glean-core/rlb/src/lib.rs

```diff
@@ -519,10 +521,9 @@ pub(crate) fn destroy_glean(clear_stores: bool) {
     }
 }
 
+/// TEST ONLY FUNCTION.
 /// Resets the Glean state and triggers init again.
-#[cfg(test)]
```

Comment:
Why is this not `cfg(test)` anymore?

Reply:
Because when writing tests in m-c, we need to reset the Glean state as well (other bindings have fixtures that they offer).